bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Text.java

```diff
@@ -2514,7 +2514,8 @@ int untranslateOffset (int offset) {
 @Override
 void updateMenuLocation (Event event) {
 	Point point = display.mapInPixels (this, null, getCaretLocationInPixels ());
```

Comment:
Nit: I would rename this one to `pointInPixels` so it's easier to see that you're adding pixels with pixels 2 lines below in the call to `event.setLocation(...)`